### PR TITLE
[KV] Adding new info on metric/analytics API

### PR DESCRIPTION
**Pull request by Oxyjun**
DO NOT MERGE

### Summary

Migrating PR from before the dev-docs migration. This relates to https://github.com/cloudflare/cloudflare-docs/pull/15918

Adding:

1. New chapter on "Metrics and analytics" under a new "Observability" navigation tab.
2. "Changelog" under "Platform" navigation tab.
3. "kv.yaml" file where changelogs live.

Note: 

This PR is high-priority. Iterative improvements can be made in future pushes.

No material changes have been made to this PR regarding content - only the required syntax has been changed to conform the PR to meet docs standards.

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Review by kodster28 — APPROVED** [review]

**Comment by thomasgauvin** [comment]
This PR should not be merged until we are ready to announce KV analytics

File: src/content/changelogs/kv.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+productLink: "/kv/"
+productArea: Developer platform
+productAreaLink: /workers/platform/changelog/platform/
+entries:
+
+- publish_date: "2024-08-08"
+  title: New KV Analytics API
+  description: |-
+
+    Workers KV now has a new [metrics dashboard](/kv/observability/metrics-analytics/#view-metrics-in-the-dashboard) and [analytics API](/kv/observability/metrics-analytics/#query-via-the-graphql-api) that leverages the [GraphQL Analytics API](/analytics/graphql-api/) used by many other Cloudflare products. The new analytics API provides per-account and per-namespace metrics for both operations and storage, including latency metrics for read and write operations to Workers KV.c
```
Comment (thomasgauvin) [inline src/content/changelogs/kv.yaml]:
```suggestion
    Workers KV now has a new [metrics dashboard](/kv/observability/metrics-analytics/#view-metrics-in-the-dashboard) and [analytics API](/kv/observability/metrics-analytics/#query-via-the-graphql-api) that leverages the [GraphQL Analytics API](/analytics/graphql-api/) used by many other Cloudflare products. The new analytics API provides per-account and per-namespace metrics for both operations and storage, including latency metrics for read and write operations to Workers KV.
```

File: src/content/docs/kv/observability/metrics-analytics.mdx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    "accountTag":"<YOUR_ACCOUNT_ID>",
+    "namespaceId": "<YOUR_KV_NAMESPACE_ID>",
+    "date_geq": "2024-07-15",
+    "date_leq": "2024-07-30"
+}
+```
+
+#### Operations
+
+To query the sum of read, write, delete, and list operations for a given `namespaceId` and for a given date range (`date_geq` and `date_leq`), grouped by `date` and `action`:
```
Comment (thomasgauvin) [inline src/content/docs/kv/observability/metrics-analytics.mdx]:
```suggestion
To query the sum of read, write, delete, and list operations for a given `namespaceId` and for a given date range (`date_geq` and `date_leq`), grouped by `date` and `actionType`:
```

File: src/content/docs/kv/observability/metrics-analytics.mdx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+                filter: { namespaceId: $namespaceId, date_geq: $date_geq, date_leq: $date_leq }
+                limit: 10000
+				orderBy: [date_DESC]
+            ) {
+                sum {
+                    requests
+                }
+                dimensions {
+                    date
+                    action
```
Comment (thomasgauvin) [inline src/content/docs/kv/observability/metrics-analytics.mdx]:
```suggestion
                    actionType
```

File: src/content/docs/kv/observability/metrics-analytics.mdx [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+        accounts(filter: { accountTag: $accountTag }) {
+            kvOperationsAdaptiveGroups(
+                filter: { namespaceId: $namespaceId, date_geq: $date_geq, date_leq: $date_leq, action: "GET" }
+                limit: 10000
+            ) {
+                sum {
+                    requests
+                }
+                dimensions {
+                    action
```
Comment (thomasgauvin) [inline src/content/docs/kv/observability/metrics-analytics.mdx]:
```suggestion
                    actionType
```

File: src/content/docs/kv/observability/metrics-analytics.mdx [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+
+To query the distribution of the latency for read operations for a given `namespaceId` within a given date range (`date_geq`, `date_leq`):
+
+```graphql
+query {
+    viewer {
+        accounts(filter: { accountTag: $accountTag }) {
+            kvOperationsAdaptiveGroups(
+                filter: { namespaceId: $namespaceId, date_geq: $date_geq, date_leq: $date_leq, action: "GET" }
```
Comment (thomasgauvin) [inline src/content/docs/kv/observability/metrics-analytics.mdx]:
```suggestion
                filter: { namespaceId: $namespaceId, date_geq: $date_geq, date_leq: $date_leq, actionType: "read" }
```

File: src/content/docs/kv/observability/metrics-analytics.mdx [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+```graphql
+query {
+    viewer {
+        accounts(filter: { accountTag: $accountTag }) {
+            kvOperationsAdaptiveGroups(filter: { date_geq: $date_geq, date_leq: $date_leq  }, limit: 10000) {
+                sum {
+                    requests
+                }
+                dimensions {
+                    action
```
Comment (thomasgauvin) [inline src/content/docs/kv/observability/metrics-analytics.mdx]:
```suggestion
                    actionType
```

**Comment by thomasgauvin** [comment]
We will merge when the dashboard is pushed and publicly available

**Pull request merged**